api-test/internal/esignet/driver.go

```
@@ -0,0 +1,715 @@
+}
+
+func (d *Driver) completeLogin(s *session, base, authID, executionID string, fr FlowResult) FlowResult {
+	var challengeToken, assertion, flowStatus string
+	payload := map[string]any{"executionId": executionID}
+	// Boundary for dynamic OTP freshness: any OTP delivered before login started
+	// belongs to a previous flow and is ignored (the send happens later in this
+	// loop, well after this point).
+	flowStart := time.Now()
+
+	for step := 0; step < d.maxFlowSteps; step++ {
+		pj, _ := json.Marshal(payload)
```

golangci-lint (2.12.2)

[error] Error return value of `encoding/json.Marshal` is not checked: unsafe type `any` found

(errchkjson)